README.md

```diff
@@ -1,48 +1,30 @@
 # WordPress Operator
 
-A Juju charm for a Kubernetes deployment of WordPress, configurable to use a
-MySQL backend.
+A Juju charm deploying and managing WordPress on Kubernetes.
+WordPress is the world's most popular website builder, and it's free and open-source.
```

Comment:
Indico seems to have a link directed to Indico, would having a link to wordpress also help?
i.e.
```suggestion
WordPress is the world's most popular website builder, and it's free and open-source. Find out more about WordPress' features [here](https://wordpress.com/features/).
```

Reply:
I think they remove that in newer versions of the Indico charm, but adding a link to WordPress official website is a good idea. Added, thanks!